_config.yml

```diff
@@ -16,6 +16,7 @@
 version: 0.7.0
 url: "https://ontoportal.github.io" # the base hostname & protocol for your site
 baseimgs: "/documentation/assets/images/ontoportal"
+figures_link: "/documentation/assets/images/figures"
```

Comment:
baseimgs is a link to a folder named "ontoportal" inside the assets.
but the figures are in a another folder named figures.
And why I didn't use the ontoportal folder to put figures?
because we'll have another folder named ontoportal for the screenshot of ontoportal, so this will be confusing .